### PR TITLE
Update PF templates with port

### PR DESCRIPTION
**Pull request by FokkeZB**
For #247 and https://github.com/TheThingsNetwork/docs/issues/66

### CODE REVIEW

File: ttnctl/cmd/applications_pf.go [hunk 1]
```diff
@@ -20,9 +20,9 @@ converting and validating binary payload.`,
   INFO Connecting with Handler...
   INFO Found Application
   INFO Decoder function
-function Decoder(bytes) {
+function Decoder(bytes, port) {
   var decoded = {};
-  decoded.led = bytes[0];
+  if (port === 1) decoded.led = bytes[0];
```
Comment (johanstokking) [inline ttnctl/cmd/applications_pf.go]: Use braces

File: ttnctl/cmd/applications_pf_set.go [hunk 2]
```diff
@@ -23,20 +23,20 @@ The functions are read from the supplied file or from STDIN.`,
...
   INFO Discovering Handler...
   INFO Connecting with Handler...
-function Decoder(bytes) {
+function Decoder(bytes, port) {
   // Decode an uplink message from a buffer
   // (array) of bytes to an object of fields.
   var decoded = {};
 
-  // decoded.led = bytes[0];
+  // if (port === 1) decoded.led = bytes[0];
```
Comment (johanstokking) [inline ttnctl/cmd/applications_pf_set.go]: Idem

File: ttnctl/cmd/applications_pf_set.go [hunk 3]
```diff
@@ -23,20 +23,20 @@ The functions are read from the supplied file or from STDIN.`,
...
 
   return decoded;
 }
 ########## Write your Decoder here and end with Ctrl+D (EOF):
-function Decoder(bytes) {
+function Decoder(bytes, port) {
   var decoded = {};
 
-  decoded.led = bytes[0];
+  if (port === 1) decoded.led = bytes[0];
```
Comment (johanstokking) [inline ttnctl/cmd/applications_pf_set.go]: Idem

File: ttnctl/cmd/applications_pf.go (hunk 1 above)
Comment (FokkeZB) [inline ttnctl/cmd/applications_pf.go]: I tried to keep it compact? :)

**Comment by FokkeZB** [comment]
Addressed request

**Pull request merged**